Plaid API Endpoints Collection.json

```diff
@@ -78,8 +143,8 @@
 									]
 								}
 							},
-							"_postman_previewlanguage": null,
-							"header": null,
```

Comment:
I don't know why all these changes occurred, happy to roll these particular ones back